core/src/main/java/greencity/controller/ManagementOrderController.java

```diff
@@ -967,4 +968,25 @@ public ResponseEntity<AddBonusesToUserDto> addBonusesToUser(@PathVariable(name =
         return ResponseEntity.status(HttpStatus.CREATED)
             .body(ubsManagementService.addBonusesToUser(addBonusesToUserDto, orderId, principal.getName()));
     }
+
+    /**
+     * Controller for get order cancellation reason.
+     *
+     * @return {@link OrderCancellationReasonDto}.
+     * @author Kharchenko Volodymyr
+     */
+    @ApiOperation(value = "Get order cancellation reason")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK, response = OrderCancellationReasonDto.class),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
```

Comment:
I see that we can have next responses:

- `200` - success
- `401` - unauthorized user
- `403` - used does not have the required role
- `404` - order with `orderId` does not exist

When can we have `400` response?